README.md

```diff
@@ -20,37 +20,16 @@ Please join the discussion on the Angle Community [Discord Server](https://disco
+Angle is a decentralized stablecoins protocol composed of smart contracts mostly deployed on the Ethereum blockchain. 
 
-Angle is the first over-collateralized, decentralized and capital-efficient stablecoin protocol. It offers full convertibility between stable assets and collateral at oracle value: you can swap 1 of collateral against 1 of stablecoin and conversely.
-
-This makes the protocol both capital efficient and highly liquid.
+It has three modules, or set of smart contracts: [Angle Core module](/concepts/overview.md), the new [Angle Borrowing module](/new-module/README.md), and the [Angle Governance module](/governance/angle-dao.md). 
 
 ### 🛣️ Roadmap
 
-Angle Protocol could be used to issue any stablecoin. and has started on mainnet with agEUR, a Euro stablecoin. Besides creating the first liquid Euro stablecoin, the goal of Angle is to create stablecoins for other types of assets.
+Angle Protocol could be used to issue any stablecoin. and has started on mainnet with agEUR, a Euro stablecoin. Besides creating the first liquid and decentralized Euro stablecoin, the goal of Angle is to create stablecoins for other types of assets.
 
 Stable CHF, GBP, JPY and KRW should be following agEUR.
 
-### 🎨 Protocol Design
-
-The protocol involves 3 agents, very common in other DeFi protocols, which all bring something to Angle, while benefitting from the protocol:
-
-- **Stable Seekers/Holders, or Users:** they can swap collateral against stable assets, and conversely swap stable assets against a whitelisted collateral of their choice at oracle value, with no slippage and small transaction fees.
-- **Hedging Agents (HAs):** they can open on-chain leveraged positions on Angle. They need to bring accepted collateral to the protocol, and are able to take positions on perpetuals futures on the available collateral/stablecoins pairs. By doing so, they insure the protocol against the volatility of the collateral brought by stable seekers. This volatility is transferred to these traders, and the protocol is able to ensure users of the convertibility of the stablecoins they own even in case of collateral price drops.
-- **Standard Liquidity Providers (SLP)** : they lend collateral to the protocol in return of a share of minting and burning fees, and of the rewards earned from investing part of the protocol reserves into yield-earning strategies.
-
-In short, Angle matches people who want stability (stable seekers) and people who want volatility (Hedging Agents). Yet, there isn't always a perfect match between supply and demand of volatility, meaning that the protocol's collateral may not be fully covered at all times by Hedging Agents. The additional collateral brought by Standard Liquidity Providers, serve as a buffer in this marketplace.
-
-The protocol is implemented as a set of **smart contracts** on top of the Ethereum blockchain.
-
-## ❓ [FAQ](faq.md)
-
-Check Frequently Asked Questions for a deeper introduction to Angle and its key features.
```

Comment:
Can make a Global FAQ, like about different things in the protocol: agEUR, Core Module, Borrowing Module, veANGLE and so on...
There are elements which are relevant to many places

Reply:
would be cool to have a global FAQ

Reply:
We can link it here, same for the glossary